pynet/datasets/euaims.py

```diff
@@ -0,0 +1,840 @@
+# -*- coding: utf-8 -*-
+########################################################################
+# NSAp - Copyright (C) CEA, 2019
+# Distributed under the terms of the CeCILL-B license, as published by
+# the CEA-CNRS-INRIA. Refer to the LICENSE file or to
+# http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html
+# for details.
+########################################################################
+
+"""
+Module provides functions to prepare different datasets from EUAIMS.
+"""
+
+# Imports
+import os
+import json
+import time
+import urllib
+import shutil
+import pickle
+import requests
+import logging
+import numpy as np
+from collections import namedtuple
+import pandas as pd
+import sklearn
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler, OneHotEncoder
+from sklearn.linear_model import LinearRegression
+from pynet.datasets import Fetchers
+from neurocombat_sklearn import CombatModel as fortin_combat
+from nibabel.freesurfer.mghformat import load as surface_loader
+
+# Global parameters
+Item = namedtuple("Item", ["input_path", "output_path", "metadata_path",
```

Comment:
A Fetcher need to return an Item.

Reply:
I had the impression that not every fetcher did. But I can make it work no prob

Reply:
Please use an Item for the output.

Reply:
Is it okay if the tuple names are not the same as these ones? Because I have a train and a test path for the input and metadata. The output path is the same as the input path